<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWatcherWidget</class>
 <widget class="QWidget" name="RegWatcherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>35</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>35</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RegWatcherWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Name</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblType">
     <property name="toolTip">
      <string>Type</string>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblAddress">
     <property name="toolTip">
      <string>Address</string>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblValue">
     <property name="toolTip">
      <string>Value</string>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
